esti/rollback_test.go

```diff
@@ -77,10 +77,12 @@ func TestHardReset(t *testing.T) {
 		"failed to commit changes repo %s branch %s", repo, mainBranch)
 
 	// commit again so we have something to revert
-	_, err = client.CommitWithResponse(ctx, repo, mainBranch, &apigen.CommitParams{}, apigen.CommitJSONRequestBody{
+	commitResp2, err := client.CommitWithResponse(ctx, repo, mainBranch, &apigen.CommitParams{}, apigen.CommitJSONRequestBody{
 		Message:    "another commit",
 		AllowEmpty: swag.Bool(true),
 	})
+	require.NoError(t, err, "failed to commit changes")
+	require.NotNil(t, commitResp2.JSON201, "failed to commit changes")
```

Comment:
Could we instead
```suggestion
	require.NoError(t, verifyResponse(commitResp2.HTTPResponse, commmitResp.Body), "failed to commitmore changes")
```
?

Reply:
Sure - for me checking JSON201 for non-nil check valid response and content type.

Reply:
Yeah!  Just that verifyResponse will give a better error message than "failed to commit changes".

Reply:
Fixed, also added message format fix.